src/assessments/seo/KeywordDensityAssessment.js

```diff
@@ -325,6 +352,135 @@ class KeywordDensityAssessment extends Assessment {
 		};
 	}
 
+	/**
+	 * Returns the score for the keyword density (for Recalibration).
+	 *
+	 * @param {Jed} i18n The object used for translations.
+	 *
+	 * @returns {Object} The object with calculated score and resultText.
+	 */
+	calculateResultRecalibration( i18n ) {
+		if ( this.hasNoMatches() ) {
+			return {
+				score: this._config.scores.underMinimum,
+				resultText: i18n.sprintf(
+					/* Translators:
+					%1$s and %4$s expand to links to Yoast.com,
+					%2$s expands to the anchor end tag,
+					%3$d expands to the recommended minimal number of times the keyphrase should occur in the text. */
+					i18n.dgettext(
+						"js-text-analysis",
+						"%1$sKeyphrase density%2$s: The focus keyword was found 0 times. " +
+						"That's less than the recommended minimum of %3$d times for a text of this length." +
+						"%4$sFocus on your keyphrase%2$s!",
+					),
+					this._config.urlTitle,
+					"</a>",
+					this._minRecommendedKeywordCount,
+					this._config.urlCallToAction,
+				),
+			};
+		}
+
+		if ( this.hasTooFewMatches() ) {
+			return {
+				score: this._config.scores.underMinimum,
+				resultText: i18n.sprintf(
+					/* Translators:
+					%1$s and %4$s expand to links to Yoast.com,
+					%2$s expands to the anchor end tag,
+					%3$d expands to the recommended minimal number of times the keyphrase should occur in the text,
+					%5$d expands to the number of times the keyphrase occurred in the text. */
+					i18n.dngettext(
+						"js-text-analysis",
+						"%1$sKeyphrase density%2$s: The focus keyword was found %5$d time. That's less than the " +
```

Comment:
All instances of `focus keyword` in the text should be changed to `keyphrase`.

Reply:
Fixed.